example/src/main/tut/typeclass/Debug.md

```diff
@@ -1,11 +1,11 @@
 ---
 layout: docs
-title:  "Show"
+title:  "Debug"
 ---
 
-# Show
+# Debug
 
-`Show` presents a safe and explicit alternative to `Object.toString`.
+`Debug` presents a safe and explicit alternative to `Object.toString`.
 By providing instances of this typeclass, a type explicitly defines that it can be converted into human-readable output.
 
```

Comment:
I think you could explicitly qualify that this typeclass isn't to be used for general showing around of arbitrary values.

Reply:
I think the "human-readable output" is pretty good.